adhocteam/Head-Start-TTADP
# TTAHUB-170/Total Number of Hrs and Grantees Graph

## Description of change

Add a new graph to the Dashboard called 'Total Hours of TTA and Number of Grants Served'.

**Files for Review**

**- Frontend**
frontend/src/widgets/TotalHrsAndGranteeGraph.js
frontend/src/widgets/TotalHrsAndGranteeGraph.css
frontend/src/widgets/__tests__/TotalHrsAndGranteeGraph.js
frontend/src/pages/Dashboard/index.js
**-Backend**
src/widgets/totalHrsAndGranteeGraph.js
src/widgets/totalHrsAndGranteeGraph.test.js
src/widgets/index.js


## How to test

Login and go to the Dashboard page you should see the new graph. Changing the date range should update the graph accordingly. If the date is less than or equal to 31 days the x-axis should display days not months.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-170


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

~- [ ] Staging smoke test completed~

### After merge/deploy
